src/Common/src/Microsoft/Win32/SafeHandles/SafeProvHandle.cs

```diff
@@ -0,0 +1,126 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Somehow I missed that these files went into src/Common/src/Microsoft/Win32/SafeHandles (I thought they were under Interop/Windows/advapi32)

I'd move them to the advapi32 directory, which is (one of) the style used by sspicli and winsock